pretix_sepadebit/signals.py

```diff
@@ -59,6 +68,70 @@ def register_csv(sender, **kwargs):
     return DebitList
 
 
+@receiver(register_mail_placeholders, dispatch_uid="payment_sepadebit_placeholders")
+def register_mail_renderers(sender, **kwargs):
+
+    ph = [SimpleFunctionalMailTextPlaceholder(
+          'due_date', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.due.date, sample=date.today()
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'account', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.info_data.get('account', " "), sample="Max Mustermann"
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'iban', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.info_data.get('iban')[0:4] + " **** " + sepadebit_payment.info_data.get('iban')[-4:], sample="DE02 **** 2051"
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'bic', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.info_data.get('bic'), sample="BYLADEM1001"
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'reference',
+            ['sepadebit_payment'],
+            lambda sepadebit_payment: sepadebit_payment.info_data.get('reference'),
+            lambda event: f'{event.settings.reference_prefix + "-" if event.settings.reference_prefix else ""}{event.slug.upper()}-XXXXXXX'
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'creditor_id', ['sepadebit_payment', "event"], lambda sepadebit_payment, event: event.settings.creditor_id, sample="DE98ZZZ09999999999"
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'creditor_name', ['sepadebit_payment', "event"], lambda sepadebit_payment, event: event.settings.creditor_name, sample="DE98ZZZ09999999999"
+        )]
```

Comment:
I think we should add ``debit_amount`` and ``debit_amount_with_currency``. The payment amount isn't guaranteed to be equal to the order total